Taskfile.yml

```diff
@@ -0,0 +1,232 @@
+version: '3'
+
+tasks:
+
+  create-network:
+    desc: Create the external network
+    cmds:
+      - (docker network create external-net) || true
+
+  init-backend:
+    desc: Initialize the backend project
+    dir: tdrs-backend
+    cmds:
+      - task: create-network
+      - docker-compose -f docker-compose.yml up -d --build
+      - docker-compose -f docker-compose.yml exec web sh -c "python ./manage.py makemigrations"
+      - docker-compose -f docker-compose.yml exec web sh -c "python ./manage.py migrate"
+      - docker-compose -f docker-compose.yml down
+      - task: sentry-down
+
+  clone-sentry-repo:
+    desc: Clone the sentry repo
+    dir: sentry
+    cmds:
+      - git clone https://github.com/getsentry/self-hosted.git || true
+
+
+  create-sentry:
+    desc: Create Sentry service
+    dir: sentry
+    cmds:
+      # limiting the memory to 2GB and CPU to only one cpu @0, for faster response, you can remove the limittask :  --cpuset-cpus 0 
+      - (docker run --privileged -p 9001:9000 -d --memory="8g" --memory-swap="20g" --name sentry docker:dind) || true
```

Comment:
Sentry would not work for me unless I gave it 12GB of memory. Im not even sure if that will be enough for our use case in the deployed environments. We don't know how Sentry's memory scales as the load on the application fails. Also, Sentry was hardly giving me any metrics with respect to Django and Postgres. Is that because I need to configure other things?

Reply:
As far as the metrics, I haven't added any of the metrics yet. I wanted to discuss this during office hour to see which route we would like to take